packages/salesforcedx-vscode-apex/test/vscode-integration/commands/forceApexExecute.test.ts

```diff
@@ -193,9 +194,8 @@ describe('Force Apex Execute', () => {
       );
 
       await executor.run({ type: 'CONTINUE', data: {} });
-
+      expect(showChannelOutputStub.notCalled).to.be.true;
```

Comment:
Shouldn't this be `calledOnce` instead of `notCalled` now that showing the channel output is set to true at the commandlet level ?

Reply:
We're also calling `showChannelOutput` at the commandlet level now as part of the `execute()` method. This test calls the `run()` method directly instead of calling `execute()`, so `showChannelOutput` shouldn't get called at this level